python_bindings/setup.py

```diff
@@ -79,18 +79,19 @@ class BuildExt(build_ext):
     """A custom build extension for adding compiler-specific options."""
     c_opts = {
         'msvc': ['/EHsc', '/openmp', '/O2'],
-        'unix': ['-O3'],
+        'unix': ['-O3', '-msse4.2'],
```

Comment:
Ohh, I see. There are other issues with the Windows built, but I have to postpone working on this. BTW, building things on MacOS lead to a discovery of a broken test and a bug in the Mean function. It can generate NaN for empty arrays. I am fixing this right now.